lib/board.rb

```diff
@@ -34,6 +34,20 @@ def valid_placement?(ship, coordinates)
       false
     end
   end
+
+  def place(ship, coordinates)
+    if valid_placement?(ship, coordinates) #moved this up, if valid_placement?(ship, coordinates) then allow rest of method to run
+       coordinates.each do |coordinate|#each method is being called on the coordinates array(from cell class)
+         @cell = @cells[coordinate] #creating local variable for the cell object(@cells[coordinate])
+        @cell.place_ship(ship) #call place_ship method(from cell class) on variable we created for cell object, pass in ship argument
```

Comment:
place_ship method came from cell class. took forever to realize you can't pull arugment name from another class with the method name.  finally got to pass using ship as argument